docs/application/tizen-studio/native-tools/ui-builder/component-attributes.md

```diff
@@ -783,7 +783,7 @@ The following attributes are used for the layout component.
 
 ### List (![List](./media/component_attributes_list_icon.png))
 
-A list component (in [mobile](../../native/guides/ui/efl/component-list-m.md) and [wearable](../../native/guides/ui/efl/component-list-w.md) applications) is a very simple list for managing a small number of items. If you need to manage a lot of items, use the genlist component instead.
+A list component (in [mobile](../../native/guides/ui/efl/mobile/component-list.md) and [wearable](../../native/guides/ui/efl/wearable/component-list.md) applications) is a very simple list for managing a small number of items. If you need to manage a lot of items, use the genlist component instead.
```

Comment:
It looks `../../../native/guides/ui/efl/mobile/component-list.md` and `../../../native/guides/ui/efl/wearable/component-list.md` are correct

Reply:
done